IdentityCore/src/cache/MSIDMacKeychainTokenCache.m

```diff
@@ -318,12 +320,14 @@ - (BOOL)saveAccount:(MSIDAccountCacheItem *)account
     NSMutableDictionary *query = [self primaryAccountAttributesForKey:key];
     NSMutableDictionary *update = [self secondaryAccountAttributesForKey:key];
     update[(id)kSecValueData] = jsonData;
+    MSID_LOG_VERBOSE(context, @"SecItemUpdate: query=%@ update=%@", query, update);
```

Comment:
Note that this might contain PII. Can you make sure you don't log PII in non PII log macroses?

Reply:
Updated to use the MSID_LOG_VERBOSE_PII macro.